### PR TITLE
Add a chart for Stack2Slack

### PR DESCRIPTION
**Pull request by dunglas**
[Stack to Slack](https://github.com/dunglas/stack2slack) is a Slack that bot monitors StackExchange tags and automatically publishes new questions in configured Slack channels.

### CODE REVIEW

File: stable/stack2slack/values.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Required
+slackApiToken: ''
+debug: '0'
+tagToChannel:
+  #api-platform.com: stackoverflow
+stackSite: 'stackoverflow'
+image:
+  repository: dunglas/stack2slack
+  tag: latest
```
Comment (unguiculus) [inline stable/stack2slack/values.yaml]: Please use a release version.

File: stable/stack2slack/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+publishes new questions in configured Slack channels.
+
+## Introduction
+
+This chart adds a Stack to Slack deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh)
+package manager.
+
+## Prerequisites
+
+- Kubernetes 1.2+ with Beta APIs enabled
```
Comment (unguiculus) [inline stable/stack2slack/README.md]: 1.7+

File: stable/stack2slack/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: A Slack bot to monitor StackOverflow/StackExchange tags
+name: stack2slack
+version: 0.1.0
+sources:
+- https://github.com/dunglas/stack2slack
+maintainers:
+- name: Kévin Dunglas
```
Comment (unguiculus) [inline stable/stack2slack/Chart.yaml]: Use Github username.

File: stable/stack2slack/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: A Slack bot to monitor StackOverflow/StackExchange tags
+name: stack2slack
+version: 0.1.0
```
Comment (unguiculus) [inline stable/stack2slack/Chart.yaml:4]: Add `appVersion`

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. Note that we've started namespacing templates (see #1785). Please apply this to your chart.

**Comment by unguiculus** [comment]
Please rebase your PR. It is way behind the current master.

**Comment by unguiculus** [comment]
Marking this as stale. Please update within one week.

**Comment by dunglas** [comment]
I'll do (sorry about the delay).

**Comment by unguiculus** [comment]
Please fix linting issue.

**Comment by unguiculus** [comment]
Please have a look at the failing CI.

**Comment by dunglas** [comment]
The CI failure looks legit. Providing a secret is mandatory to run the bot. What the procedure in this case @unguiculus?

File: stable/stack2slack/templates/_helpers.tpl [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```
Comment (unguiculus) [inline stable/stack2slack/templates/_helpers.tpl:1]: Helm 2.8+ creates a much better `_helpers.tpl`. Please update.

File: stable/stack2slack/templates/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
```
Comment (unguiculus) [inline stable/stack2slack/templates/deployment.yaml:1]: `apps/v1beta2`

File: stable/stack2slack/templates/deployment.yaml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+kind: Deployment
+metadata:
+  name: {{ template "stack2slack.fullname" . }}
+  labels:
+    app: {{ template "stack2slack.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
```
Comment (unguiculus) [inline stable/stack2slack/templates/deployment.yaml:10]: Add `spec.selector` (required with `apps/v1beta2`)

File: stable/stack2slack/templates/deployment.yaml [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+      labels:
+        app: {{ template "stack2slack.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: SLACK_API_TOKEN
```
Comment (unguiculus) [inline stable/stack2slack/templates/deployment.yaml:22]: Please use consistent list indentation style throughout the chart. I'd suggest you always indent two spaces.

File: stable/stack2slack/templates/secrets.yaml [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: {{ template "stack2slack.fullname" . }}
+  labels:
+    app: {{ template "stack2slack.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+ slackApiToken : {{ default "MISSING" .Values.slackApiToken | b64enc | quote }}
```
Comment (unguiculus) [inline stable/stack2slack/templates/secrets.yaml:12]: Incorrect indentation.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Your chart can only be merged when CI is green. In order for this to achieve you need to conditionally create the deployment similar to what you do in the NOTES.txt.

**Pull request closed without merging**